Base/Python/slicer/cli.py

```diff
@@ -68,7 +68,7 @@ def run(module, node = None, parameters = None, wait_for_completion = False, del
   delete_temporary_files: remove temp files created during execution (True by default)
   update_display: show output nodes after completion
   """
-  import slicer.util
+  #import slicer.util
```

Comment:
Because the matching snippet is also commented:

https://github.com/Slicer/Slicer/blob/d14f18f386f83943d6717cedb5d18f1ce423cada/Base/Python/slicer/cli.py#L87-L92

Reply:
Could you move the commented out import and move down to be with the commented out block below that uses it? Or simply remove the dead code?

Reply:
Will do so in in next few hours, along with updating the style guide.